tdrs-backend/tdpservice/test/test_clamav.py

```diff
@@ -15,53 +18,27 @@ def clamav_url():
     return settings.AV_SCAN_URL
 
 
-@pytest.fixture
-def fake_file_name():
```

Comment:
Moved these fixtures to the root `conftest` so they can be available across multiple modules (needed in next PR).